lib/util/db.js

```diff
@@ -88,6 +88,10 @@ const resultCount = (result) => Number(result[0].count);
 ////////////////////////////////////////////////////////////////////////////////
 // ERROR HANDLING
 
+// deals with postgres returning eg '"xmlFormId", version' error text.
+const dequote = (text) => ((text.startsWith('"') && text.endsWith('"')) ? text.slice(1, -1) : text);
+const splitPostgresTuple = (text) => text.split(/,\s*/g);
```

Comment:
What happens if an element of the tuple itself contains a comma? (Does Postgres add quotes around such elements?)

Reply:
Also, if the element of a tuple contains leading white space, will `\s*` remove that? Maybe we could use a single space rather than `\s*`? (Or maybe it doesn't matter if the white space disappears?)

Reply:
If form IDs can't contain commas, and versions must be numeric, I guess these comments wouldn't affect xmlFormId/version tuples. But maybe they'd affect other Postgres tuples?

Reply:
Short of writing a lexer, I can't account for those cases. Postgres will inconsistently quote/notquote things in various cases so I'd have to actually walk the string to make this completely airtight. I think there are no problem cases possible yet, and even if there are, it's just error message output.

Reply:
Sounds good, and we can always circle back to this if we run into an issue.